### PR TITLE
7 sdx 8a meican user signup

### PR DESCRIPTION
**Pull request by lmarinve**
starting helping to review and approve changes

### CODE REVIEW

**Review by lmarinve — COMMENTED** [review]
good, is now using gmail!

**Review by lmarinve — COMMENTED** [review]
good

**Review by lmarinve — COMMENTED** [review]
reviewed

File: modules/circuits/views/nodes/nodes/nodes.php
Comment (lmarinve) [inline modules/circuits/views/nodes/nodes/nodes.php:1]: reviewed

File: modules/circuits/views/nodes/status/status.php
Comment (lmarinve) [inline modules/circuits/views/nodes/status/status.php:1]: reviewed

File: modules/circuits/views/nodes/view/graph.php
Comment (lmarinve) [inline modules/circuits/views/nodes/view/graph.php:1]: reviewed

File: modules/circuits/views/nodes/view/view.php
Comment (lmarinve) [inline modules/circuits/views/nodes/view/view.php:1]: reviewed

**Review by lmarinve — COMMENTED** [review]
ok

**Review by usmanazFIU — APPROVED** [review]
ok

**Pull request merged**